notebooks/Mitigating Disparities in Ranking from Binary Data.ipynb

```diff
@@ -20,14 +21,36 @@
+    "\n",
+    "* **Domain:**\n",
+    "  * Semi-synthetic domain based on law-school admission data. Please review the usage notes at the end of [Overview](#Overview).\n",
+    "\n",
+    "* **Task description:**\n",
+    "  * Assessment and mitigation of unfairness.\n",
+    "    The data has binary labels, which are used to train probabilistic score predictors and classifiers.\n",
+    "\n",
+    "* **Metrics:**\n",
+    "  * area under ROC curve\n",
+    "  * worst-case area under ROC curve\n",
+    "  * balanced accuracy\n",
+    "  * demographic parity difference (for both binary and continuous predictions)\n",
+    "\n",
+    "* **Algorithms:**\n",
+    "  * `fairlearn.reductions.ExponentiatedGradient`\n",
+    "  * `fairlearn.reductions.GridSearch`\n",
+    "  * `fairlearn.postprocessing.ThresholdOptimizer`\n",
+    "\n",
+    "## Overview\n",
     "\n",
     "We consider the task of ranking students for admission to law school using the data collected in [Law School Admissions Council's (LSAC) National Longitudinal Bar Passage Study](https://eric.ed.gov/?id=ED469370); specifically, the version downloaded from [Project SEAPHE](http://www.seaphe.org/databases.php). We highlight some of the fairness considerations that come up not only in school admissions, but also in other ranking scenarios. Necessarily, our example is simplified and ignores many real-world considerations specific to school admissions.\n",
     "\n",
-    "The data set contains information about law students collected by LSAC between 1991 and 1997. Some of the information is available at the admission time (such as the undergraduate GPA and LSAT score), and some describes the performance of the students once admitted. We also have access to their self-identified race. Throughout this exercise, we will limit the attention to those self-identified as **black** and **white** (two largest groups).\n",
+    "The data set contains information about law students collected by LSAC between 1991 and 1997. Some of the information is available at the admission time (such as the undergraduate GPA and LSAT score), and some describes the performance of the students once admitted. We also have access to their self-identified race. To simplify this example, we will limit the attention to those self-identified as **black** and **white** (two largest groups) and restrict our attention to two features (undergraduate GPA and LSAT score).\n",
+    "\n",
+    "To help with ranking law school applicants, we train a model that uses the information that is available about a student at the admission time to predict the probability that they will pass their bar exam. The predictions of our model are intended to be used (among other factors) by admission officers to select the applicants. After training the initial model, we examine differences in the predictions it induces across two the two groups. We then mitigate these differences using the `GridSearch` approach in Fairlearn.\n",
```

Comment:
Don't we use ExpGrad and Postprocessing too?

Reply:
good point. fixed!